lib/src/Pos/Util/Servant.hs

```diff
@@ -520,11 +526,13 @@ instance ( HasServer (subApi :> res) ctx
          HasLoggingServer config (apiType a :> res) ctx where
     routeWithLog = paramRouteWithLog
 
-instance HasLoggingServer config res ctx =>
+instance {-# OVERLAPPING #-}
+         HasLoggingServer config res ctx =>
          HasLoggingServer config (Summary s :> res) ctx where
     routeWithLog = inRouteServer @(Summary s :> LoggingApiRec config res) route identity
 
-instance HasLoggingServer config res ctx =>
+instance {-# OVERLAPPING #-}
+         HasLoggingServer config res ctx =>
          HasLoggingServer config (Description d :> res) ctx where
     routeWithLog = inRouteServer @(Description d :> LoggingApiRec config res) route identity
 
```

Comment:
You managed to get around this overlapping instances problem? Was it just to do with polykinds and GHC 804?

Reply:
Yes. It was just the PolyKinds requirement that GHC 804 put up, and the resulting overlapping issue. I'm still unsure as to why earlier GHCs did to neither flag for PolyKinds nor have issues with the overlapping instances.